backend/app/rest/tags_routes.py

```diff
@@ -18,3 +18,26 @@ def get_tags():
     except Exception as e:
         error_message = getattr(e, "message", None)
         return jsonify({"error": (error_message if error_message else str(e))}), 500
+    
+
+@blueprint.route("/delete/<int:tag_id>", methods=["PUT"], strict_slashes=False)
```

Comment:
So you technically did do this right with the PUT (since we're not actually "deleting" anything), but this should be changed into a DELETE route for now. The plan is to remove the `status` field in a follow up PR which I talked about at our last work session, so I think it's more convenient to do it now than later.

Also for the route name, you can keep it consistent with what's been done for other DELETE routes, for example:
```
@blueprint.route("/<int:delete_id>", methods=["DELETE"], strict_slashes=False)
@require_authorization_by_role({"Relief Staff", "Regular Staff", "Admin"})
def delete_log_record(delete_id):
```